lib/inactive_users_suspension_reminder.rb

```diff
@@ -19,14 +20,18 @@ def send_reminders
         UserMailer.suspension_reminder(user, @days_to_suspension).deliver_now
         Rails.logger.info "#{self.class}: Successfully sent email to #{user.email}."
       rescue *ERRORS_TO_RETRY_ON => e
+        notify_airbrake(e, user)
         Rails.logger.debug "#{self.class}: #{e.class} - #{e.message} while sending email to #{user.email} during attempt (#{(tries..3).count}/3)."
         sleep(3) and retry if (tries -= 1) > 0
 
         Rails.logger.warn "#{self.class}: Failed to send suspension reminder email to #{user.email}."
-        notify_airbrake(e, user)
```

Comment:
This changes the notification behaviour here.  Perviously it would only notify Errbit when it had given up retrying. This is desireable, otherwise the Errbit notifications can get quite noisy.